core/src/main/java/apoc/schema/Schemas.java

```diff
@@ -185,6 +187,18 @@ private AssertSchemaResult createUniqueConstraint(Schema schema, String lbl, Str
         return result;
     }
 
+    private Object getLabelForAssert(IndexDefinition definition, boolean nodeIndex) {
```

Comment:
what about using `List<String>` as a return type, and return either the list as is (multi-label / multi-type), or a list of 1 element (single label / single type)?

Reply:
This would cause a breaking change,
because I would have as field `label`, no longer `"Foo"` but `["Foo"]` for example.
I don't know if it's worth it, wdyt?